libraries/core-react/src/Button/Button.jsx

```diff
@@ -84,17 +79,17 @@ const Base = ({ base, baseDisabled: disabled }) => {
   `
 }
 
-const ButtonBase = styled.button.attrs(({ type = 'button' }) => ({
-  type,
-}))`
+const ButtonBase = styled.button`
   margin: 0;
```

Comment:
Suggest we remove this margin as we have gotten feedback about margins not working on button

Reply:
Added custom margin test, was success 🏆

Reply:
👏